x-pack/plugins/security_solution/public/detections/pages/alert_details/components/error_page.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiCode, EuiEmptyPrompt } from '@elastic/eui';
+import { ERROR_PAGE_TITLE, ERROR_PAGE_BODY } from '../translations';
+
+export const AlertDetailsErrorPage = memo(({ eventId }: { eventId: string }) => {
```

Comment:
This is just a simple error page where the user is told their `_id` doesn't point to a document. Alternatively, can use a not found page here